src/backlight/portfolio/portfolio.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+import numpy as np
+from typing import List
+
+from backlight.positions.positions import Positions
+from backlight.datasource.marketdata import MarketData
+from backlight.metrics.position_metrics import calc_pl
+
+
+class Portfolio:
+    """
+    An abstract definition of a Portfolio is defined as a list of Positions
+    Each element of the list represent the positions of an asset that contribute to the portfolio
+    """
+
+    def __init__(self, positions: List[Positions]):
+        self.portfolio = positions
+        self.pl = pd.DataFrame()
+
+    @property
+    def value(self) -> pd.DataFrame:
+        """ DataFrame of the portfolio valuation of each asset"""
+        for asset_positions in self.portfolio:
+            # Compute PL of positions of each asset
+            self.pl[asset_positions.symbol] = calc_pl(asset_positions)
+        return self.pl
+
+    # Normalize PL to one asset reference and sum
+    def normalized_total_pl(
```

Comment:
Why do you add `normalized_total_pl` as a method? You can choose function or method here.

Reply:
FYI
https://stackoverflow.com/questions/8108688/in-python-when-should-i-use-a-function-instead-of-a-method

We can find better materials, but I just share it.

Reply:
I make separated functions.